wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/model/impl/devfile/ObjectConverter.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.api.workspace.server.model.impl.devfile;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import javax.persistence.AttributeConverter;
+import javax.persistence.Converter;
+
+@Converter(autoApply = true)
+public class ObjectConverter implements AttributeConverter<Object, String> {
+
+  ObjectMapper mapper = new ObjectMapper();
+
+  @Override
+  public String convertToDatabaseColumn(Object attribute) {
+    try {
+      return mapper.writeValueAsString(attribute);
+    } catch (JsonProcessingException e) {
+      e.printStackTrace();
+    }
+    return null;
+  }
+
+  @Override
+  public Object convertToEntityAttribute(String dbData) {
+    if (Boolean.TRUE.toString().equals(dbData) || Boolean.FALSE.toString().equals(dbData)) {
+      return Boolean.getBoolean(dbData);
+    } else {
+      try {
+        return Integer.parseInt(dbData);
```

Comment:
please don't control flow with exceptions. It's nasty and it's performance killer. We have apache-commons-lang https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/math/NumberUtils.html#isCreatable-java.lang.String-